acorn/acorn-tests.ts

```diff
@@ -33,7 +33,9 @@ const parser = new acorn.Parser({}, 'export default ""', 0);
 
 const node = new acorn.Node(parser, 1, 1);
 
-acorn.addLooseExports(function () {}, parser, {});
+acorn.addLooseExports(function () {
+    return {} as ESTree.Program
```

Comment:
I'd prefer a real test node - it's not that hard to construct a minimal real `Program` node.